feature-libs/checkout/components/components/review-submit/review-submit.component.ts

```diff
@@ -47,7 +49,9 @@ export class ReviewSubmitComponent {
     protected checkoutStepService: CheckoutStepService,
     protected paymentTypeService: PaymentTypeFacade,
     protected checkoutCostCenterService: CheckoutCostCenterFacade,
-    protected userCostCenterService: UserCostCenterService
+    protected userCostCenterService: UserCostCenterService,
+    // TODO:#13888 remove after the deprecation is done
+    protected featureConfigService?: FeatureConfigService
```

Comment:
Same question as in `feature-libs/checkout/components/components/delivery-mode/delivery-mode.component.ts`